komodor/integrations.go

```diff
@@ -0,0 +1,64 @@
+package komodor
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+const IntegrationsUrl string = DefaultEndpoint + "/integrations/kubernetes"
+
+type Kubernetes struct {
+	Id          string `json:"apiKey"`
+	ClusterName string `json:"clusterName"`
+}
```

Comment:
I think you can remove the `ClusterName` from the struct. You are setting it but never use it

Reply:
The `ClusterName` property is used in both Get and Create methods.